app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -103,6 +105,39 @@ def add_collections
       end
     end
 
+    # TODO: documentation
+    def sanitize_qa_uri_values!
+      LOCAL_QA_FIELDS.each do |field|
+        next if parsed_metadata[field].blank?
+
+        parsed_metadata[field].each_with_index do |value, i|
+          next if value.blank?
+
+          if value.match?(::URI::DEFAULT_PARSER.make_regexp)
+            value = value.strip.chomp.sub('https', 'http')
```

Comment:
Maybe a second value in the initializer or create a hash for which ones need to be converted to http.

Reply:
Is there possibly a more programmatic solution? What if we checked the service for both http and https versions and then did a sub accordingly? @sephirothkod

Reply:
Oh I see, by config option you mean declaring whether they should ALL be http or ALL be https?

Reply:
checking the service is probably the best way to go. check the raw link, if it doesn't exist in the QA do something like:
```
if value.contain?("https")
  value = value.sub('https', 'http')
else
  value = value.sub('http', 'https')
end
```
then try again. error if it still doesn't exist in the QA

Reply:
you might be able to do a ternary of the above.